docs/security/using-fail2ban-for-security.md

```diff
@@ -126,11 +126,14 @@ Fail2ban reads its configuration files so that all `.conf` files are read first
 
         cd /etc/fail2ban
 
-2.  Copy `fail2ban.conf` to `fail2ban.local`:
+2.  The file `fail2ban.conf` contains the default configuration variables for Fail2ban logging, the socket used to communicate with the daemon, and the location of the PID file. Changes should be made in a separate file, `fail2ban.local`, which overrides `fail2ban.conf`. If you want, you can copy `fail2ban.conf` to `fail2ban.conf`, commenting out all variables, and then uncomment only the options you want to modify:
 
         cp fail2ban.conf fail2ban.local
+        sed -i 's/^/#/g' fail2ban.local
```

Comment:
@yarikoptic Here's the above simplified into one command, matching only the definitions and not the section name:

    sed 's/\(^[[:alpha:]]\)/# \1/' fail2ban.conf | sudo tee fail2ban.local 1&> /dev/null

Could you clarify what you mean by the issue of stale configs across releases? I'm not sure that needs to be addressed here since users should update (or at least check) their .local file when they upgrade to a new release. Any changes they've already made to an outdated .local file won't easily be updated without a custom script or something that goes beyond the scope of a one-line command.  I might be misunderstanding your point though.